components/reports/demographics/DemographicsReport.vue

```diff
@@ -1,43 +1,72 @@
 <template>
   <div>
-    <h3 class="title is-3">Demographics</h3>
+    <h3 class="title is-3">Demographics and Health</h3>
+    <div class="content my-5 is-size-4">
+      <p>
+        The demographic and health data below have been curated from publicly
+        available sources, including data from the U.S. Census and the CDC’s
+        Behavioral Risk Factor Surveillance System (BRFSS), to make them more
+        accessible to Alaska communities. This resource was developed in
+        collaboration with community and state partners to ensure it meets local
+        needs. Our goal is to break down barriers to data access, providing key
+        information at the community level where decisions are made.
+      </p>
+      <p>
+        Health data are only shown for locations with an adult population of 50
+        or more, based on the 2020 U.S. Census.
+      </p>
+    </div>
     <DemographicsMap />
-    <div class="content is-size-4">
+    <div class="content is-size-4 mt-6">
       <p>
-        ⚠️ Demographic data is presented only regionally, at the level of
-        boroughs or census tracts.
+        Demographic data is presented only regionally, at the level of boroughs,
+        census-sesignated places (CDPs), incorporated places, and census tracts.
       </p>
       <p>
-        In this section,
-        <strong>{{ place }} is used as the name for {{ commentEdited }}</strong
-        >, as shown in the map above.
+        <strong>This section shows information from {{ commentEdited }}</strong
```

Comment:
```suggestion
        <strong>This section shows {{ commentEdited }}</strong
```

This is to fix the following:
![image](https://github.com/user-attachments/assets/a7c7c1ad-be99-460e-91ff-973ce51e49d1)